tests/foreman/api/test_contentview.py

```diff
@@ -802,6 +803,29 @@ def test_positive_promote_composite_multiple_content_multiple(self):
             len(composite_cv.version[0].read().environment),
         )
 
+    @tier2
+    def test_positive_promote_out_of_sequence(self):
+        """Try to publish content view few times in a row and then re-promote
+        first version to default environment
+
+        @Assert: Content view promoted out of sequence properly
+
+        @Feature: ContentView
+        """
+        content_view = entities.ContentView(organization=self.org).create()
+        for _ in range(REPEAT):
+            content_view.publish()
+        content_view = content_view.read()
+        self.assertEqual(len(content_view.version), REPEAT)
+        self.assertEqual(len(content_view.version[0].read().environment), 0)
+        lce_list = content_view.version[REPEAT - 1].read().environment
+        self.assertEqual(len(lce_list), 1)
+        promote(content_view.version[0], lce_list[0].id, force=True)
+        content_view = content_view.read()
+        self.assertEqual(len(content_view.version[0].read().environment), 1)
+        self.assertEqual(
+            len(content_view.version[REPEAT - 1].read().environment), 0)
+
```

Comment:
Can you add some comments to these assert statements, it is difficult to understand what is being asserted without the comments.